src/OtpInput/OtpInput.tsx

```diff
@@ -5,7 +5,7 @@ import { OtpInputProps, OtpInputRef } from "./OtpInput.types";
 import { VerticalStick } from "./VerticalStick";
 import { useOtpInput } from "./useOtpInput";
 
-export const OtpInput = forwardRef<OtpInputRef, OtpInputProps>((props, ref) => {
+export const OtpInput = forwardRef<OtpInputRef, OtpInputProps>((props: OtpInputProps, ref) => {
```

Comment:
No need for this change; props types getting populated from `forwardRef<>` generics